app/src/main/java/com/lastaoutdoor/lasta/viewmodel/ConversationViewModel.kt

```diff
@@ -78,8 +82,15 @@ constructor(
   fun send(message: String) {
     viewModelScope.launch {
       if (message.isNotEmpty()) {
-        repository.sendMessage(userId, friendUserId, message)
-        updateConversation()
+        try {

```

Comment:
This will beneficiate the person who implements the ErrorManager (because it's me, I thank you for that)